src/idl_gen_lobster.cpp

```diff
@@ -181,7 +195,7 @@ class LobsterGenerator : public BaseGenerator {
     }
     if (IsVector(field.value.type)) {
       code += def +
-              "_length():\n        return "
+              "_length()->int:\n        return "
```

Comment:
The changed codegen will have an effect on sample/test code. Run `scripts/generate_code.py` and add changed files to this PR. Run `tests/lobstertest.lobster` to verify it works.